test/tools/d_do_test.d

```diff
@@ -554,6 +557,103 @@ bool collectExtraSources (in string input_dir, in string output_dir, in string[]
     return true;
 }
 
+/++
+Applies custom transformations defined in transformOutput to testOutput.
+
+Currently the following actions are supported:
+ * "sanitize_json"       = replace compiler/plattform specific data from generated JSON
+ * "remove_lines(<str>)" = remove all lines containing <str>
+
+Params:
+    transformOutput = list of transformation identifiers
+    testOutput      = the existing output to be modified
+++/
+void applyOutputTransformations(string transformOutput, ref string testOutput)
```

Comment:
I would reverse the oder of the args so that when used in UFCS it reads `testOutput. applyOutputTransformations(transformOutput)`